charts/datadog/templates/system-probe-configmap.yaml

```
@@ -32,8 +32,12 @@ data:
       collect_dns_stats: {{ $.Values.datadog.systemProbe.collectDNSStats }}
       max_tracked_connections: {{ $.Values.datadog.systemProbe.maxTrackedConnections }}
       conntrack_max_state_size: {{ $.Values.datadog.systemProbe.conntrackMaxStateSize }}
+      {{- if eq  (include "enableRuntimeCompiler-is-set" .) "true" }}
       enable_runtime_compiler: {{ $.Values.datadog.systemProbe.enableRuntimeCompiler }}
+      {{- end }}
+      {{- if eq  (include "enableKernelHeaderDownload-is-set" .) "true" }}
       enable_kernel_header_download: {{ $.Values.datadog.systemProbe.enableKernelHeaderDownload }}
```

instead of defaulting `enableKernelHeaderDownload` to `nil` can we  keep `true` as default value but do something like:
```suggestion
      {{- if not $.Values.datadog.systemProbe.enableKernelHeaderDownload }}
      enable_kernel_header_download: false
      {{- end }}
```